app/.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
```

Comment:
does eslint support extending config, so we don't have to repeat this in two files? cc @pdehaan